src/CallHandler.js

```diff
@@ -444,6 +429,40 @@ async function _startCallApp(roomId, type) {
     });
 }
 
+function _terminateCallApp(roomId) {
+    Modal.createTrackedDialog('Confirm Jitsi Terminate', '', QuestionDialog, {
+        hasCancelButton: true,
+        title: _t("End conference"),
+        description: _t("Ending the conference will end the call for everyone. Continue?"),
+        button: _t("End conference"),
+        onFinished: (proceed) => {
+            if (!proceed) return;
+
+            // We'll just obliterate them all. There should only ever be one, but might as well
+            // be safe.
+            const roomInfo = WidgetStore.instance.getRoom(roomId);
+            const jitsiWidgets = roomInfo.widgets.filter(w => WidgetType.JITSI.matches(w.type));
+            jitsiWidgets.forEach(w => {
+                // setting invalid content removes it
+                WidgetUtils.setRoomWidget(roomId, w.id);
+            });
+        },
+    });
+}
+
+function _hangupWithCallApp(roomId) {
```

Comment:
Maybe just "hangup"...?

```suggestion
function _hangupCallApp(roomId) {
```

Reply:
applied manually due to renaming